index.js

```diff
@@ -57,6 +63,37 @@ app.get('/products/:Id/description-and-standards', (req, res) => {
   });
 });
 
+app.delete('/products/:Id', (req, res) => {
+  const productId = req.params.Id;
+  con.query(`DELETE FROM StandardsandDescriptions WHERE Product_id = ${productId}`, (SandDDeleteErr) => {
+    if (SandDDeleteErr) throw SandDDeleteErr;
```

Comment:
- can use try/catch block

Reply:
- can handle both errors differently if we need diff behvaoir for diff error types
- (in each cb) or instantiate new error obj and give a message

Reply:
respond to client with every request regardless of success or error